superset/datasets/commands/importers/v0.py

```diff
@@ -301,9 +302,23 @@ def __init__(
     def run(self) -> None:
         self.validate()
 
+        # TODO (betodealmeida): add rollback in case of error
         for file_name, config in self._configs.items():
             logger.info("Importing dataset from file %s", file_name)
-            import_from_dict(db.session, config, sync=self.sync)
+            if isinstance(config, dict):
+                import_from_dict(db.session, config, sync=self.sync)
+            else:  # list
+                for dataset in config:
+                    # UI exports don't have the database metadata, so we assume
+                    # the DB exists and has the same name
+                    params = json.loads(dataset["params"])
+                    database = (
+                        db.session.query(Database)
+                        .filter_by(database_name=params["database_name"])
+                        .one()
+                    )
+                    dataset["database_id"] = database.id
```

Comment:
do you want a guard on this if database is null?

Reply:
The method `.one()` raises an exception if the database doesn't exist (as opposed to `.first()`, which returns `Optional[Database]`.

We don't capture the exception if the database doesn't exist, but for now I'm keeping the behavior consistent with the old import. Once we have these unit tests merged I want to refactor the v0 import to use the same foundation as v1 (`ImportMixin` and better validation).

Reply:
sounds good. Thanks for the explanation!